central/apitoken/service/service_impl_test.go

```diff
@@ -0,0 +1,53 @@
+package service
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/central/role"
+	roleMock "github.com/stackrox/rox/central/role/datastore/mocks"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/grpc/authn"
+	"github.com/stackrox/rox/pkg/grpc/authn/mocks"
+	"github.com/stackrox/rox/pkg/testutils/roletest"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func TestServiceImpl_ListAllowedRoles_SortsRoleAlphabetically(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	mockIdentity := mocks.NewMockIdentity(mockCtrl)
+
+	mockDatastore := roleMock.NewMockDataStore(mockCtrl)
+	roleOne := roletest.NewResolvedRole("Analyst", map[string]storage.Access{},
+		role.AccessScopeIncludeAll,
+	)
+
+	roleTwo := roletest.NewResolvedRole("Admin", map[string]storage.Access{},
+		role.AccessScopeIncludeAll,
+	)
+
+	roleThree := roletest.NewResolvedRole("Writer", map[string]storage.Access{},
```

Comment:
I would probably put `Writer` as the first role to make it more obvious that no sorting has been done yet.